packages/formspree-react/src/ValidationError.tsx

```diff
@@ -1,27 +1,27 @@
-import React from 'react';
-import type { FormError } from '@formspree/core';
+import React, { type ComponentPropsWithoutRef } from 'react';
+import type { FieldValues, SubmissionErrorResult } from '@formspree/core';
 
-export type ValidationErrorProps = {
+export type ValidationErrorProps<T extends FieldValues = FieldValues> = {
+  errors: SubmissionErrorResult<T> | null;
+  field?: keyof T;
   prefix?: string;
-  field?: string;
-  errors: FormError[];
-  [x: string]: any;
-};
+} & ComponentPropsWithoutRef<'div'>;
 
 export const ValidationError: React.FC<ValidationErrorProps> = (props) => {
```

Comment:
We should rename this since it's not only for validation. It's server errors. react-hook-form calls a similar component `ErrorMessage`.